packages/cashier-v2/package.json

```diff
@@ -0,0 +1,49 @@
+{
+    "name": "@deriv/cashier-v2",
+    "private": true,
+    "version": "1.0.0",
+    "main": "dist/cashier-v2/js/index.js",
+    "engines": {
+        "node": "18.x"
+    },
+    "scripts": {
+        "build": "rimraf dist && NODE_OPTIONS='-r ts-node/register' webpack --progress --config \"./webpack.config.ts\"",
```

Comment:
nit: Why is webpack config file in Typescript? 👀

Reply:
Its ok to have it in TS right? As it can help in intellisense.
Even account-v2 is using TS for all their config files.

Reply:
Yeah I just found out webpack accepts Typescript configuration, I think its fine then cuz you can have the type safety